autogen/versions.tf.tmpl

```diff
@@ -26,6 +26,10 @@ terraform {
       source  = "hashicorp/google-beta"
       version = ">= 4.40, < 5.0"
     }
+    random = {
+      source  = "hashicorp/random"
+      version = "~> 3.4"
```

Comment:
Let's relax this constraint if possible to reduce the likelyhood of a breaking change.  This is what we have been using for other modules that didn't previously include a version for random:

```suggestion
      version = ">= 2.1"
```

Reply:
done